### DISCUSSION TITLE
Dedicated servers don't show if they are password protected

### DISCUSSION THREAD

**Discussion by Ydrec** [FakeFishGames/Barotrauma#15366] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While using server browser, all dedicated servers show up as passwordless, but when you try to join them you do get password prompt (if server actually requires password)

### Reproduction steps

Open serverbrowser
Filter by not having password
Attempt to join dedicated server with password protection but which shows up as passwordless
Get password prompt

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/d0b9685462d6e6c66dc3843912f6d00997b15038 (commit in our private repo). Fix coming up in a future update.
